src/components/comments/container/commentsContainer.tsx

```diff
@@ -221,6 +224,10 @@ const CommentsContainer = ({
         setLComments(filteredComments);
       } else {
         filteredComments = propComments.filter(_applyFilter);
+        console.log('filteredComments : ', filteredComments);
+        if (postType === PostTypes.WAVE) {
+          deleteWaveMutation.mutate(filteredComments);
```

Comment:
possibly we can make a callback in wavesScreen and handle wave delete inside instance of useWavesQuery